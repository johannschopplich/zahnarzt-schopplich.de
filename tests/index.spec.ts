import { test, describe, expect } from "vitest";
import Impressum from "../src/pages/Impressum.mdx";

describe("imprint page", () => {
  test("stub default layout", () => {
    expect(Impressum.title).toMatchSnapshot();
  });
});
