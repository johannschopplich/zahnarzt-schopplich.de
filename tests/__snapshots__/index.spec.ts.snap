// Vitest Snapshot v1

exports[`index page > stub default layout 1`] = `"Impressum"`;
